database/index.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require ('mongoose');
+mongoose.connect('mongodb://localhost/houses', {} , (err)=>{
+  if (err) {
+    console.log(err);
+  }
+  console.log('im connected?');
+});
+
+let repoSchema = mongoose.Schema({
```

Comment:
This is a vague variable name, I would recommend something more specific. What would happen if you had to use two schemas?

Also this should be a const _not_ at let